src/components/tasks/card/index.tsx

```diff
@@ -28,15 +32,31 @@ const Card: FC<Props> = ({
   const [assigneeProfilePic, setAssigneeProfilePic] = useState(
     `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${cardDetails.assignee}/img.png`,
   );
+  const { SUCCESS, ERROR } = ToastTypes;
   const isUserAuthorized = useContext(isUserAuthorizedContext);
+  const [taskTagLevel, setTaskTagLevel] = useState<taskItem[]>()
   const [showEditButton, setShowEditButton] = useState(false);
   const [keyLongPressed] = useKeyLongPressed();
+  const [isLoading, setIsloading] = useState<boolean>(false);
+  const ITEM_BY_ID_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/items/byitem/${cardDetails.id}`;
+  const ITEMS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/items`;
```

Comment:
Could we not get from one constants or `api.js` file?

Reply:
moved these to `urls.js` file